File: 8KYU/Job Matching #1/solution.js
function match(candidate, job) {
  if(!candidate.minSalary || !job.maxSalary)
    throw "error";
  
  var wiggle = candidate.minSalary * 0.1;
  if(candidate.minSalary - wiggle > job.maxSalary)
    return false;
  return true;
}

File: 8KYU/Job Matching #1/solution.cs
using StriveObjects;
using System;

public class Strive
{
  public static bool Match(Candidate c, Job j)
  {
    if(j.MaxSalary == null || c.MinSalary == null)
      throw new Exception("argh");
    return (j.MaxSalary >= (c.MinSalary * 0.9));
  }
}